pom.xml

```diff
@@ -38,15 +38,16 @@
   </developers>
 
   <properties>
-    <revision>1.14.10</revision>
+    <revision>1.15.6</revision>
```

Comment:
Not sure what you did here but I think you botched this PR.

In general I would strongly discourage use of destructive Git operations inside PR branches. Limit yourself to `push`ing commits, and let the maintainer decide whether to squash merge or whatever. If you have merge conflicts, simply `git merge master` or similar and push the merge commit. Among other things, using the Git history graph in a straightforward way allows GitHub to behave intelligently, for example letting reviewers look at only recently added commits.